qiita_pet/templates/study_base.html

```diff
@@ -112,11 +112,37 @@
   $(document).ready(function() {
     fill_base({{study_info['study_id']}});
   });
+
+  function validate_delete_study_text() {
+    if ($("#study-alias").val() == "{{study_info['study_alias']}}") {
+      $('#delete-study-button').prop('disabled', false);
+    } else {
+      $('#delete-study-button').prop('disabled', true);
+    }
+  }
+
+function delete_study() {
+  if ($("#study-alias").val() != "{{study_info['study_alias']}}") {
+    alert("The added name doesn't match the study alias");
```

Comment:
This reads a bit weird. Change "The added name" by "The provided alias", or "The given alias", but it is not added.